src/plugins/event-plugins/WebVitalsPlugin.ts

```diff
@@ -34,28 +47,63 @@ export class WebVitalsPlugin extends InternalPlugin {
     configure(config: any): void {}
 
     protected onload(): void {
+        this.context.eventBus.subscribe(Topic.EVENT, this.handleEvent); // eslint-disable-line
```

Comment:
nit: Instead of disabling es-lint here, you can fix this warning by (A) using an arrow function to declare `handleEvent`, or (B) annotating the function with `this: WebVitalsPlugin`.